examples/group/README.md

```diff
@@ -0,0 +1,182 @@
+- [glob patterns](https://docs.pachyderm.com/latest/concepts/pipeline-concepts/datum/glob-pattern/) - for "RegEx-like" string matching on file paths and names.
+
+You might also want to brush up your [datum](https://docs.pachyderm.com/latest/concepts/pipeline-concepts/datum/relationship-between-datums/) knowledge. 
+
+***Pre-requisite***
+- A workspace on [Pachyderm Hub](https://docs.pachyderm.com/latest/pachhub/pachhub_getting_started/) (recommended) or Pachyderm running [locally](https://docs.pachyderm.com/latest/getting_started/local_installation/).
+- [pachctl command-line ](https://docs.pachyderm.com/latest/getting_started/local_installation/#install-pachctl) installed, and your context created (i.e., you are logged in)
+
+***Getting started***
+- Clone this repo.
+- Make sure Pachyderm is running. You should be able to connect to your Pachyderm cluster via the `pachctl` CLI. 
+Run a quick:
+```shell
+$ pachctl version
+COMPONENT           VERSION
+pachctl             1.12.0
+pachd               1.12.0
+```
+Ideally, have your pachctl and pachd versions match. At a minimum, you should always use the same major & minor versions of your pachctl and pachd. 
+
+## 2. Data structure and naming convention
+Our first example comes from a simple healthcare use case:
+- We have a list of Labresults from various tests done on Patients by given Labs. Each file contains one type of test results from a given Lab for a given Patient.
+
+The second example is derived from a simplified retail use case: 
+- Purchases and Returns are made in given Stores. 
+- Those Stores have a given location (here, a zip code). 
+- There are 0 to many Stores in a given Zipcode.
+
+Let's have a look at the data structure and naming convention of our first example.  
+* Repo: `labresults` - Our file names follows the following "-" separated pattern: 
+
+1. **T** + {Time stamp}
+2. Type of test (Here **LIPID** for all our files)
+3. **PATID** + {Patient identifier}
+4. **CLIA** + {Lab/Hospital identifier}
+
+```shell
+    └── T1606707557-LIPID-PATID1-CLIA24D9871327.txt
+    └── T1606331395-LIPID-PATID2-CLIA24D9871327.txt
+    └── T1606707613-LIPID-PATID1-CLIA24D9871328.txt
+    └──  ...
+```
+For information, here is what the content of those txt files looks like. 
+
+![labresultsample.png](./img/labresultsample.png)
+
+The second dataset (example 2) is shared with the "Join pipelines"' examples. Find the structure of the data and naming conventions [here](Link to join pipeline example // README chapter 2).
+
+## 3. Example 1 - Simple group-by pipelines 
+***Goal***
+We  want to aggregate our labresults by patient or by hospital. We will create two separate pipelines out of the same repository, one for each case.
+
+1. **Pipeline input repository**: `labresults` - Group by patient: the group will be done by PATID - Group by hospital: the group will be done by CLIA.
```

Comment:
The formatting here seems confusing.  Perhaps a bulleted list bellow the enumerated list, with markup to delineate items like CLIA and PATID, like so:

1. **Pipeline input repository**: labresults 
   * Group by _patient_: the group will be done by `PATID`
   * Group by _hospital_: the group will be done by `CLIA`.

If you adopt that formatting for  `PATID` & `CLIA` here, it might be best to do it throughout the document